Compose/TucsonTestStand/docker-compose.yml

```diff
@@ -193,6 +193,28 @@ services:
       lab-priv-network:
         ipv4_address: "10.0.100.157"
 
+  mt-hexapod-sim-1:
```

Comment:
I would suggest being explicit with the container name: mt-cam-hexapod-sim.

Reply:
good catch... I actually did not know which one was which when I did this. :)